wiki/Client/Release_stream/Lazer/Feature_comparison/en.md

```diff
@@ -0,0 +1,438 @@
+
+In lazer, not hitting the slider head will give a MISS judgement for the whole slider. After missing a slider head, combo, score, and accuracy can still be gained from slider ticks and repeats, while only score and accuracy can still be gained from the slider end.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | No |
+
+#### Slider ends do not cause hitsounds when not hit
+
+In stable, slider ends would play their hitsounds even if they were missed, as long as any part of the slider was hit. This has been changed such that hitsounds match inputs 1:1.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | Yes |
+| Intentionally changed | Yes |
+| Needs further consideration | No |
+
+#### The spinner spin speed cap of 477 RPM has been removed
+
+Instead of a speed cap, spinners now have a score cap. The reason for this is inconsistency in how score is awarded, because the algorithm is dependent on frame rate.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | Yes |
+
+#### Aspire-like glitched sliders are not supported
+
+![](img/aspire-slider.gif)
+
+Some adventurous beatmaps exploit glitches in the stable client that allowed for very weird slider mechanics. These range from zero-length sliders acting as invisible circles, to cross-screen stretched and squished sliders.
+
+More discussion and consideration will be needed for how much of Aspire beatmaps will be compatible going forward. For example, invisible circles might become a properly supported feature in the future.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | Yes |
+
+### osu!taiko
+
+#### The playfield aspect ratio is limited
+
+The osu!taiko playfield is stretched based on window size, making the apparent AR of gameplay change in line with window size. Originally this was limited completely as it affects playability but switched to only limiting the maximum aspect ratio due to community push-back for having it "as it was on stable".
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | Yes |
+| Intentionally changed | Yes |
+| Needs further consideration | Maybe |
+
+#### Notes that overlap swells cannot be hit
+
+Some gimmick maps make use of notes that overlap swells.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | No |
+| Needs further consideration | Yes |
+
+#### Drumrolls do not prevent mashing
+
+In stable, drumrolls could not be hit too quickly or too slowly. This restriction has been lifted, just like in ScoreV2.
```

Comment:
Are you sure? Looking at the implementation that does definitely not look to be the case. Each tick in the drumroll has a judgement window.

Reply:
yes, drumroll ticks have a hit window in both stable and lazer, but stable scorev1 has an additional layer that blocks input if you're hitting [more than twice as fast as ticks appear or hitting slower than every 5th tick](https://github.com/ppy/osu-wiki/pull/8245/files#diff-b82a37959e54dc656a70a58c3417fa40c6bced969b74229307f3fcc785425150R29) ([relevant stable code](https://github.com/peppy/osu-stable-reference/blob/1531237b63392e82c003c712faa028406073aa8f/osu!/GameplayElements/HitObjects/Taiko/SliderTaiko.cs#L362-L396))

drumroll ticks don't have miss windows unlike other hit objects, so in lazer (and stable scorev2) you're able to mash through the drumroll as long as some of your keypresses land on the ticks' hit windows

there is an open issue about this at https://github.com/ppy/osu/issues/5694